tests/stdcxx/enc/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ enclave_compile_options(stdcxx_enc PRIVATE
     -fno-builtin-memset
     -fno-builtin-fprintf
     -fno-builtin-printf
+    -Og -ggdb
```

Comment:
This was an accidental check-in. I turned on `snmalloc` and pretty much debugged each one of the failing tests to make sure that debugging works. I noticed that for this test (`stdcxx`) when built with clang, your are able to set a breakpoint (via `b enc_test`) but gdb is not able to map the source. When compiled with gcc, gdb works perfectly.

This accidental check-in caused LVI tests to fail (since LVI required that -g not be present) and @mingweishih helped root cause it.